components/log-viewer-webui/server/settings.json

```diff
@@ -9,5 +9,6 @@
     "MongoDbIrFilesCollectionName": "ir-files",
 
     "ClientDir": "../client/dist",
-    "IrFilesDir": "../../../build/clp-package/var/data/ir"
+    "IrFilesDir": "../../../build/clp-package/var/data/ir",
+    "YlvDir": "../yscope-log-viewer/dist"
```

Comment:
Let's also update the start-clp.py script for the config override: https://github.com/y-scope/clp/blob/921a388cfcccf465e344e103fe7956459056dc9e/components/clp-package-utils/clp_package_utils/scripts/start_clp.py#L808